### PR TITLE
Update stdci to use okd-4.1 provider

### PR DESCRIPTION
**Pull request by rwsu**
okd-4.1.0 appears to be broken. This change updates the okd default to okd-4.1 which updates to minor releases "under the hood". See https://github.com/kubevirt/kubevirtci/pull/126

Some of this PR contains changes by slintes from https://github.com/kubevirt/kubevirt/pull/2623

### CODE REVIEW

File: cluster-up/cluster/kind/common.sh [hunk 1]
```diff
@@ -14,12 +14,12 @@ function _wait_kind_up {
         sleep 10
     done
     echo "Waiting for dns to be ready ..."        
-    kubectl wait -n kube-system --timeout=12m --for=condition=Ready -l k8s-app=kube-dns pods
+    _kubectl wait -n kube-system --timeout=12m --for=condition=Ready -l k8s-app=kube-dns pods
```
Comment (djzager) [inline cluster-up/cluster/kind/common.sh]: What's going on here?

Comment (rwsu) [reply cluster-up/cluster/kind/common.sh]: There is a function below called _kubectl that sets the correct kubectl. Actually, cluster/kind/common.sh isn't used by okd as far as I can tell. I'll remove this.

**Comment by arachmani** [comment]
ci test please

**Comment by arachmani** [comment]
lgtm

**Pull request merged**